### DISCUSSION TITLE
Do I need a switch if I'm using the UDM Pro to get IPTV?

### DISCUSSION THREAD

**Discussion by ericyoshino** [fabianishere/udm-iptv#10] [Q&A] [answered]
Hi Fabian,

I have a UDM PRO and I trying to get IPTV.

I'm studying your guide before diving in.

According to the Global Design there is a switch between the UDM Pro and the Decoders.

Can't I just plug the Decoders directly in the UDM Pro regular LAN ports?

Thank you for taking the time to help the community.

**Comment by fabianishere** [comment, marked as answer]
There is no need to have a switch in between and you can just plug the decoders directly into the UDM (Pro) switch ports. The guide shows one way of setting up your network, however, the switch is not necessary there.

**Reply by ericyoshino** [reply]
Thank you Fabian!

Appreciate it
